wcfsetup/install/files/acp/templates/applicationManagement.tpl

```diff
@@ -98,8 +98,9 @@
 								<small>{$application->getPageURL()}</small>
 							</td>
 							<td class="columnText columnLandingPageID">
-								<select name="landingPageID[{$application->packageID}]">
-									<option value="0">{lang}wcf.acp.application.landingPage.default{/lang}</option>
+								<select name="landingPageID[{$application->packageID}]" required>
+									<option value="">{lang}wcf.global.noSelection{/lang}</option>
+									<option value="0"{if $application->getAbbreviation() !== 'wcf'}{if $application->landingPageID === null} selected{/if}{else} disabled{/if}>{lang}wcf.acp.application.landingPage.default{/lang}</option>
```

Comment:
```suggestion
									<option value="0"{if $application->getAbbreviation() === 'wcf'} disabled{elseif $application->landingPageID === null} selected{/if}>{lang}wcf.acp.application.landingPage.default{/lang}</option>
```

I literally had to copy this nested `if` into an editor and reformat it to double check it.